package.json

```diff
@@ -71,6 +71,46 @@
     ],
     "yarn.lock": "yarn deduplicate-packages --list --fail"
   },
+  "eslintConfig": {
+    "extends": [
+      "@qlik/eslint-config/react",
+      "@qlik/eslint-config/jest"
+    ],
+    "parserOptions": {
+      "project": "./tsconfig.json"
+    },
+    "rules": {
+      "no-param-reassign": [
+        2,
+        {
+          "props": false
+        }
+      ],
+      "@typescript-eslint/no-floating-promises": 0,
+      "@typescript-eslint/no-unused-expressions": [
+        1,
+        {
+          "allowShortCircuit": true,
+          "allowTernary": true
+        }
+      ],
+      "@typescript-eslint/unbound-method": 0,
+      "@typescript-eslint/restrict-template-expressions": 0
+    },
+    "root": true
+  },
+  "dependencies": {
+    "@material-ui/core": "4.12.4",
```

Comment:
Why are these moved to `dependencies`?

Reply:
Interesting. It was complaining before that these should be real dependencies, but not anymore it seems. I'll move them back